Misc/NEWS.d/next/Documentation/2021-11-03-14-51-03.bpo-45680.9_NTFU.rst

```diff
@@ -0,0 +1,2 @@
+Improve links to ``__class_getitem__`` and ``GenericAlias`` docs in the
+`typing` module's documentation.
```

Comment:
FYI, you can write one single news item for all 3 PRs, I will then apply `skip news` label and merge subsequent PRs without a news.

You can use the news at https://github.com/python/cpython/pull/29335/files, then delete the news for the other two.

Reply:
@Fidget-Spinner done!